docs/source/format.rst

```diff
@@ -39,6 +39,9 @@ Many of these items allow substitutions (explained below).
   fully qualified URL (with host and scheme). If not qualified the
   test builder will be responsible for determining host and scheme.
   **Required**
+* ``query_parameters``: An optional dictionary of query parameters
+  that will be added to the ``url``. If there is an existing set of
+  query parameters they wil be extended.
```

Comment:
I'm sorry that I still haven't gotten around to following up - might still take a few days though.

Nevertheless: I don't think the structure above is confusing at all, it's in line with how many libraries handle query parameters. Of course the dictionary discards order, thus `foo=hello&bar=1&bar=2` is equivalent to `bar=1&bar=2&foo=hello` - but that was deemed acceptable. (Of course order for multiple values of the same parameter is retained.)